index.js

```diff
@@ -172,19 +172,53 @@ function processWarning () {
     process.emitWarning(warning.message, warning.name, warning.code)
   }
 
+  // const emitInterface = new Proxy(emitted, {
+  //   get (target, prop, receiver) {
+  //     const val = Reflect.get(emitted, prop, emitted);
+
+  //     if (prop === 'get') {
+  //       return function (code) {
+  //         const state = emitted.get(code)
+  //         return state === STATES.UNLIMITED_ONGOING || state === STATES.LIMITED_FINAL
+  //       }
+  //     }
+  //     return Reflect.get(emitted, prop, emitted);
+  //   }
+  // })
+
   return {
     create,
     createDeprecation,
     emit,
-    emitted: {
-      get (code) {
-        const state = emitted.get(code)
-        return state === STATES.UNLIMITED_ONGOING || state === STATES.LIMITED_FINAL
-      }
-    }
+    emitted: buildProxy(emitted)
   }
 }
 
 module.exports = processWarning
 module.exports.default = processWarning
 module.exports.processWarning = processWarning
+
+/* istanbul ignore next */
```

Comment:
Why?

Reply:
instanbul does not recognize the prototype chain calls

anyway, let me simplify this crap

Reply:
Please include an exclusion comment if it needs to be kept.